{
  throw new RuntimeException("Exception test");
}
